### ISSUE TITLE
[1.15.2] The milk bucket Easter egg doesn’t work

### ISSUE THREAD

**Issue by Nevrai** [EssentialsX/Essentials#2992]
### Information

**Full output of `/ess version`**:

```
Server version: 1.15.2-R0.1-SNAPSHOT git-Paper-90 (MC: 1.15.2)
EssentialsX version: 2.17.1.54
LuckPerms version: 5.0.72
Vault is not installed. Chat and permissions may not work.
```

I don’t know why it’s telling me that Vault isn’t installed. That must also be a bug, as it’s definitely installed. I didn’t see that message when using `/ess version` until I updated to EssentialsX version 2.17.1.54. (I created a separate issue for this: #2998.)

**Server log**: https://gist.github.com/DeleteMetaInf/ca37dc8de365e7af5a906101a614d7e1

**EssentialsX config**: https://gist.github.com/DeleteMetaInf/9f516794efd1e149287027d77fa398de

#### **Description**
Right-clicking a mob with a milk bucket does not turn it into a baby even though the config option _milk-bucket-easter-egg_ is set to _true_, as it is by default, in EssentialsX’s config.yml.

#### **Steps to reproduce**
1. Right-click a mob with a milk bucket with the default EssentialsX config.

#### Expected behaviour
Right-clicking a mob should turn it into a baby.

**Comment by triagonal** [comment]
The config is slightly unclear on this - left-clicking (punching) "ageable" mobs with a milk bucket does indeed turn them into a baby, not right-clicking. Perhaps the config should be amended to prevent further confusion.

Near duplicate of #1129, which seems to only have partly fixed the issue.

**Comment by pop4959** [comment]
This has been addressed by #3028 in 8069370b8c7ae51473f7ca84a57f69425d349f5a

**Comment by iPlayNone** [comment]
left-click (punch) a mob with a milk bucket to turn it to a baby mob

it won't deal any damage to the mob
